### DISCUSSION TITLE
LUA error message without line number

### DISCUSSION THREAD

**Discussion by Benno00** [FrSkyRC/ETHOS-Feedback-Community#4340] [General]
Hello Bertrand,
I installed version 1.5.12 today, but unfortunately I get the error message
←[31m006312 bad argument #1 to 'tonumber' (value expected)←[0m
The line number where the error occurs and in which LUA is missing here, or how should I assign the error message?
Almost all of my LUAs are no longer working, what have you changed?

**Comment by Benno00** [comment]
print("read_me:",file) doesn't work anymore, why?

**Reply by bsongis-frsky** [reply]
Strange. What is the error? What is `file`?

**Comment by Benno00** [comment]
LIPO CELL Count will not work
local sensor = system.getSource(widget.source[idx][1]:name())
if sensor~=nil then
	ccells=sensor:value(OPTION_CELL_COUNT)		
	cmin=sensor:stringValue(OPTION_CELL_LOWEST)
	cmax=sensor:stringValue(OPTION_CELL_HIGHEST)
	cmin=string.gsub(cmin, "V", "")
	cmax=string.gsub(cmax, "V", "")
	deprint("Sensor Name: "..sensor:name())
	deprint("ccels: "..ccells)
	deprint("cmin: "..cmin)
	deprint("cmax: "..cmax)
end		
![image](https://github.com/user-attachments/assets/f9e9b911-d17a-4b0a-84d1-3ee0d0243756)

**Reply by bsongis-frsky** [reply]
The doc includes an example for OPTION_CELLS_COUNT (not OPTION_CELL_COUNT):
```lua
print(source:value({options=OPTION_CELLS_COUNT}))
```

**Comment by Benno00** [comment]
it works like this now, where is the DOCUMENTATION for it?
local sensor = system.getSource(widget.source[idx][1]:name())
	if sensor~=nil then
		ccells=sensor:value({options=OPTION_CELL_COUNT})		
		cmin=sensor:stringValue({options=OPTION_CELL_LOWEST})
		cmax=sensor:stringValue({options=OPTION_CELL_HIGHEST})
		cmin=string.gsub(cmin, "V", "")
		cmax=string.gsub(cmax, "V", "")
		deprint("Sensor Name: "..sensor:name())
		deprint("ccels: "..ccells)
		deprint("cmin: "..cmin)
		deprint("cmax: "..cmax)
	end		
	
![image](https://github.com/user-attachments/assets/965018f8-6cbc-4070-a908-c56f7a4def90)

**Comment by Benno00** [comment]
It would be nice if all changes to LUA were documented so that developers can adapt accordingly.
e.g. the debug function print("Data:", field) no longer works, just print("Data:"..field) why?

**Reply by bsongis-frsky** [reply]
I am sorry, but `print` is a standard function from Lua, I am not sure it has been changed recently!
Also all changes are documented, and the documentation includes more & more examples.
I know it will be never enough, but you also have full example scripts in this repository.

**Comment by bsongis-frsky** [comment]
I need your script to investigate the missing line number in errors, I think you are in an async handler, but I am not sure
